README.md

```diff
@@ -4,24 +4,39 @@
   <h1 align="center">PlaceNoter</h1>
 </p>
 
-
 - This extension replaces your new tab with a note taking app so you never have to leave chrome. (that's it for now.)
 - Download it from [chrome webstore.](https://chrome.google.com/webstore/detail/pagenoter/jefjneinemilpncgcfdglggeheiaakfc?hl=en-GB&authuser=0)
 
 If you **👍 / ❤️ what I'm doing**, consider **🌟ing the repo**, **I** and the **Open-Source-Community** appreciate it very much ❤️.
 
-
 ## Demo
 
-- You can try it out without installing the extension at https://sereneinserenade.github.io/placenoter/ 
+- You can try it out without installing the extension at https://sereneinserenade.github.io/placenoter/
 
 https://user-images.githubusercontent.com/45892659/156848685-6a8043e6-de07-4c37-9c3d-be6e65060647.mp4
 
+## Development
 
-## Stargazers
-[![Stargazers repo roster for @sereneinserenade/placenoter](https://reporoster.com/stars/dark/sereneinserenade/placenoter)](https://github.com/sereneinserenade/placenoter/stargazers)
+```
+npm i
+npm run dev # Runs on http://localhost:3000/
+```
+
+If you open the URL, you will get an error: `This localhost page can’t be found` - a kind of 404 error
+
+In order to see it running, you need to install it as a local extension. To do that, navigate to chrome://extensions/ and click the `Load unpacked` button, it will ask you to Select the extension directory":3000/
 
+It you try and open the page http://localhost:3000/ you will get an error: `This localhost page can’t be found` - a kind of 404 error.
 
+In order to see it running, you need to install it as a local extension. To do that, navigate to chrome://extensions/ and click the `Load unpacked` button, it will ask you to "Select the extension directory", so navigate to the repo root, and click open.
```

Comment:
```suggestion
In order to see it running, you need to install it as a local extension. To do that, navigate to chrome://extensions/ and click the `Load unpacked` button, it will ask you to "Select the extension directory", so navigate to the repo root, select the `dist` folder, and click open.
```